packages/release-manifests/api-report.md

```diff
@@ -0,0 +1,34 @@
+## API Report File for "@backstage/release-manifests"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+// @public
+export function getByReleaseLine(
+  options: GetByReleaseLineOptions,
+): Promise<ReleaseManifest>;
+
+// @public
+export type GetByReleaseLineOptions = {
```

Comment:
@backstage/maintainers i wonder if there's something to be said about _not_ making named types for minimal things such as this - could just as well have been inlined in the function declaration. Or should we always bite the bullet and make the type no matter what the context, in case it grows in the future?